format-specs/geoparquet.md

```diff
@@ -50,33 +50,55 @@ Each geometry column in the dataset must be included in the columns field above
 
 | Field Name |                               Type                                      |                                                                   Description                                                                     |
 | ---------- | ----------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------- |
-| crs       | string   | **REQUIRED** [WKT2](http://docs.opengeospatial.org/is/12-063r5/12-063r5.html) string representing the Coordinate Reference System (CRS) of the geometry.  |
+| crs       | string   | **REQUIRED** [WKT2](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html) string representing the Coordinate Reference System (CRS) of the geometry.  |
 | encoding | string | **REQUIRED** Name of the geometry encoding format. Currently only 'WKB' is supported. |
 
 #### crs
 
-It is strongly recommended to use [EPSG:4326 (lat, long)](https://spatialreference.org/ref/epsg/4326/) for all data, so in most cases the value of the crs should be:
+The Coordinate Reference System (CRS) is a mandatory parameter for each geometry column defined in geoparquet format. 
+
+The CRS must be provided in [WKT](https://en.wikipedia.org/wiki/Well-known_text_representation_of_coordinate_reference_systems) version 2, also known as **WKT2**. WKT2 has several revisions, this specification supports the revisions from [2015](http://docs.opengeospatial.org/is/12-063r5/12-063r5.html) and [2019](https://docs.opengeospatial.org/is/18-010r7/18-010r7.html): WKT2_2015, WKT2_2015_SIMPLIFIED, WKT2_2019, WKT_2019_SIMPLIFIED. 
+
+
+As the most common CRS for datasets is latitude/longitude, for the widest interoperability we recommend [EPSG:4326](https://spatialreference.org/ref/epsg/wgs-84) for all data, so in most cases the value of the crs should be:
```

Comment:
We can maybe link to https://epsg.org/crs_4326/WGS-84.html instead of https://spatialreference.org (which is known to not be very up to date). 
Although that official page from the EPSG database doesn't include different representations (like WKT) as spatialreference.org or epsg.io do, but having no WKT instead of outdated WKT is maybe better ..

Reply:
I think spatial reference is more popular and I don't expect someone to go there to get the WKT2 string, we already have an example in the readme for the WKT2 string.

If somebody else is happy with this change I'll add it, not a strong opinion here.

Reply:
I know spatialreference.org is more popular, but unfortunately it is also horribly outdated. Its own home page says "Last update: 2013"

Reply:
Ok, good point. Added.